root/etc/s6-overlay/s6-rc.d/svc-sidekiq/run

```diff
@@ -15,16 +15,16 @@ cd /app/www || exit 1
 if [[ ${SIDEKIQ_ONLY,,} == "true" ]] && [[ -n ${SIDEKIQ_QUEUE} ]]; then
     echo "*** Starting sidekiq handling ${SIDEKIQ_QUEUE} queue with ${SIDEKIQ_THREADS} threads ***"
     exec \
-        s6-setuidgid abc /usr/bin/bundle exec sidekiq -q "${SIDEKIQ_QUEUE}" -c ${SIDEKIQ_THREADS}
+        s6-setuidgid abc /usr/bin/bundle exec "sidekiq -q ${SIDEKIQ_QUEUE} -c ${SIDEKIQ_THREADS}"
```

Comment:
Don't quote this, SIDEKIQ_THREADS *has* to be an int, or it will fail to start.

Reply:
I moved the quotes to quote the exec instead of the param (`SIDEKIQ_THREADS`) does this still behave incorrectly?